sentry/src/routes/channel.rs

```diff
@@ -846,4 +859,195 @@ mod test {
             balances.spenders.get(&CREATOR),
         );
     }
+
+    #[tokio::test]
+    async fn get_channels_list() {
+        let mut app = setup_dummy_app().await;
+        app.config.channels_find_limit = 2;
+        let init_chain = GANACHE_1337.clone();
+
+        let mut config = GANACHE_CONFIG.clone();
+
+        // Assert that the Ganache chain exist in the configuration
+        let mut config_chain = config
+            .chains
+            .values_mut()
+            .find(|chain_info| chain_info.chain.chain_id == init_chain.chain_id)
+            .expect("Should find Ganache chain in the configuration");
+
+        // override the chain to use the outpace & sweeper addresses that were just deployed
+        config_chain.chain = init_chain.clone();
```

Comment:
You don't need this... GANACHE_1337 load the value from GANACHE_CONFIG, please check source.
If you need to get the token too, you can use the GANACHE_INFO_1337 instead. GANACHE_INFO_1337.tokens["Mocked Token 2"]